app/routes/agenda-items.ts

```diff
@@ -135,35 +140,45 @@ export default class AgendaItemsRoute extends Route {
     if (
       model?.agendaItems?.toArray().length > 0 &&
       params.keyword === this.keywordStore.keyword &&
-      params.municipality === this.municipality &&
+      params.municipalityLabels === this.municipalityLabels &&
       params.sort === this.sort &&
       params.plannedStartMin === this.plannedStartMin &&
       params.plannedStartMax === this.plannedStartMax
     ) {
       return model;
     }
     this.keywordStore.keyword = params.keyword || "";
-    this.municipality = params.municipality || "";
+    this.municipalityLabels = params.municipalityLabels || "";
     this.sort = params.sort || "";
     this.plannedStartMin = params.plannedStartMin || "";
     this.plannedStartMax = params.plannedStartMax || "";
 
+    /*
```

Comment:
**nitpick:** Can you remove commented code in order to keep code clean 🙏

Reply:
Done!